Makefile

```diff
@@ -25,8 +28,17 @@ var/lib/grafana/%.json: src/%.jsonnet
 	mkdir -p $(shell dirname $@)
 	bin/jsonnet -o /$@ $<
 
+var/lib/grafana/plugins:
+	mkdir -p ./var/lib/grafana/plugins
+
+var/lib/grafana/plugins/grafana-groupedbarchart-panel-master:
+	curl -Lo /tmp/grafana_bar_chart_plugin.zip $(BAR_CHART_PLUGIN)
+	rm -rf ./var/lib/grafana/plugins/*
+	unzip /tmp/grafana_bar_chart_plugin.zip -d ./var/lib/grafana/plugins/
```

Comment:
Maybe we can put this in a plugins section

Reply:
Yes) Thanks) Added the section)